src/tools/http_speed.js

```diff
@@ -40,6 +42,17 @@ const http_agent = argv.ssl ?
     new https.Agent({ keepAlive: true }) :
     new http.Agent({ keepAlive: true });
 
+const buffers_pool_sem = new Semaphore(1024*1024*1024, {
```

Comment:
I think linter is getting upset here.
```suggestion
const buffers_pool_sem = new Semaphore(1024 * 1024 * 1024, {
```